Makefile

```diff
@@ -13,7 +13,7 @@ endif
 include $(ONL)/make/config.mk
 
 # All available architectures.
-ALL_ARCHES := amd64 powerpc armel arm64
+ALL_ARCHES := amd64 armel
```

Comment:
I'm sure this was a temporary local change for your development but you cannot submit this change.

Reply:
Yes, that was my local change.. Didn't know that it was part of the commit, will revert it back. Thanks for pointing it out.